src/util/Xhr.js

```diff
@@ -58,13 +65,8 @@ class Xhr {
         this.sharedLinkPassword = sharedLinkPassword;
         this.axios = axios.create();
 
-        if (typeof responseInterceptor === 'function') {
```

Comment:
@priyajeet axios will do nothing if you pass in null or a non function, so i'm avoiding checking the type of 4 things.